doc: added manual test plan for auto orgs [IDE-1507]

acke opened the pull request:
### Description

Manual test plan for IntelliJ

### Checklist

- [ ] Read and understood the [Code of Conduct](https://github.com/snyk/snyk-intellij-plugin/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/snyk/snyk-intellij-plugin/blob/master/CONTRIBUTING.md).
- [ ] Tests added and all succeed
- [ ] Linted
- [ ] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### Screenshots / GIFs

_Visuals that may help the reviewer. Please add screenshots for any UI change. GIFs are most welcome!_

acke committed:
doc/added manual test plan for auto orgs

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/snyk/snyk-intellij-plugin?pullRequest=739) <br/>All committers have signed the CLA.

acke committed:
doc: add note that test plan has been moved to Confluence [IDE-1507]

The manual test plan for IDE-1366 has been moved to Confluence for better collaboration and tracking. Added a note at the top of the file pointing to the Confluence location.
